### PR TITLE
FAI-643: Trusty UI: Error screens and messages

### PR DESCRIPTION
**Pull request by kelvah**
See: https://issues.redhat.com/browse/FAI-643

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

File: ui-packages/packages/trusty/src/components/Templates/NotFound/NotFound.tsx [hunk 1]
```diff
@@ -17,20 +17,18 @@ const NotFound = () => {
...
+            Page not found
           </Title>
         </TextContent>
       </PageSection>
       <PageSection isFilled={true}>
         <EmptyState variant={'xl'}>
           <EmptyStateIcon icon={WarningTriangleIcon} />
           <Title size="2xl" headingLevel="h4">
-            The page you are looking for does not exist
+            The page you’re looking for doesn’t exist
```
Comment (manstis) [inline ui-packages/packages/trusty/src/components/Templates/NotFound/NotFound.tsx:28]: It's no big deal but this isn't an apostrophe, but a "U+2019 : RIGHT SINGLE QUOTATION MARK"

"U+0027 : APOSTROPHE" would probably be the _usual_ "go to".

**Review by manstis — APPROVED** [review]
How picky do you want me to be? Approving..

File: ui-packages/packages/trusty/src/components/Templates/NotFound/NotFound.tsx (hunk 1 above)
Comment (kelvah) [reply ui-packages/packages/trusty/src/components/Templates/NotFound/NotFound.tsx:28]: This is interesting.
[According to wikipedia](https://en.wikipedia.org/wiki/Apostrophe#Unicode) the right single quotation mark should be a better choice for contractions.

I saw in the past word processors automatically convert the prime mark, the flat apostrophe, to the right quot mark.

Comment (manstis) [reply ui-packages/packages/trusty/src/components/Templates/NotFound/NotFound.tsx:28]: Oee that is interesting. I live and learn.

**Review by sjamboro — APPROVED** [review]

**Pull request merged**